tests/contrib/flask/__init__.py

```diff
@@ -7,13 +8,36 @@
 from tests.utils import TracerTestCase
 
 
+class DDFlaskTestClient(FlaskClient):
+    def __init__(self, *args, **kwargs):
+        super(DDFlaskTestClient, self).__init__(*args, **kwargs)
+
+    def open(self, *args, **kwargs):
+        res = super(DDFlaskTestClient, self).open(*args, **kwargs)
+        if hasattr(res, "close"):
```

Comment:
We can use make_sequence instead and the failing test will pass. I am not sure why tho https://github.com/pallets/werkzeug/blob/b1911cd0a054f92fa83302cdb520d19449c0b87b/src/werkzeug/wrappers/response.py#L378